manifests/service.pp

```diff
@@ -1,6 +1,13 @@
 class drbd::service {
+
+  if $drbd::service_ensure == 'unmanaged' {
+    $_ensure = undef
+  } else {
+    $_ensure = $drbd::service_ensure
+  }
+
   @service { 'drbd':
-    ensure  => running,
+    ensure  => $_ensure,
```

Comment:
Do you have an actual usecase for this? It feels strange to define a service but not manage the state

Reply:
If you manage ressources with some clustermanager like https://www.linbit.com/en/pacemaker/, it **could** be possible that you want to have the service ensure to be set to undef - so not managed by puppet.  
Otherwise puppet would start the resource with every puppetrun and the clustermanager stopps it again.